pkg/collector/worker/worker.go

```diff
@@ -148,7 +149,7 @@ func (w *Worker) Run() {
 		if err != nil {
 			log.Errorf("Error getting default sender: %v. Not sending status check for %s", err, check)
 		}
-		serviceCheckTags := []string{fmt.Sprintf("check:%s", check.String())}
+		serviceCheckTags := []string{fmt.Sprintf("check:%s", check.String()), "dd_enable_check_intake:true"}
```

Comment:
I'm curious, what is the purpose if this new tag?

Reply:
There is backend work to stop processing this metric. The tag will be used for customers that choose to opt-in to keeping this metric enabled so we can differentiate the traffic from "explicitly opt-in" usage vs old agents sending this metric by default.